test/integration/testArrayBind.js

```diff
@@ -636,14 +660,16 @@ describe('Test Array Bind Force Error on Upload file', function ()
   });
 });
 
-describe('testArrayBind - full path with cancel', function ()
+describe('Test Array Bind - full path with cancel', function ()
 {
   let connection;
+  const DATABASE_NAME = connOption.valid.database;
+  const SCHEMA_NAME = connOption.valid.schema;
   const tableName = `${DATABASE_NAME}.${SCHEMA_NAME}.testAB`;
 
   const rowsToInsert = 10;
-  const delayPerRowMs = 5000;
-  const cancelInsertAfterMs = 2000;
+  const delayPerRowMs = 10000;
```

Comment:
Can we put comment why we increase these timinings?

Reply:
I have added some explanation